Strategic/Map Screen Interface Bottom.cpp

```diff
@@ -697,7 +697,7 @@ void BtnTimeCompressMoreMapScreenCallback( GUI_BUTTON *btn,INT32 reason )
 			fMapScreenBottomDirty = TRUE;
 		}
 
-	btn->uiFlags|=(BUTTON_CLICKED_ON);
+		btn->uiFlags|=(BUTTON_CLICKED_ON);

```

Comment:
Avoid mixing formatting changes with code changes. [Separate functional and non-functional changes.](https://mtlynch.io/code-review-love/#6-separate-functional-and-non-functional-changes)